src/uploadSarif.ts

```diff
@@ -8,26 +8,30 @@ import { promisify } from "util";
 
 import * as utils from "./util/utils";
 
+export async function zipFile(file: string): Promise<string> {
+    const fileContents = await fs.readFile(file, "utf-8");
+    ghCore.debug(`Raw upload size: ${utils.convertToHumanFileSize(fileContents.length)}`);
+    const zippedFile = (await promisify(zlib.gzip)(fileContents)).toString("base64");
+    ghCore.debug(`Zipped file: ${zippedFile}`);
+    ghCore.info(`Zipped upload size: ${utils.convertToHumanFileSize(zippedFile.length)}`);
+
+    return zippedFile;
+}
+
 export async function uploadSarifFile(
-    ghToken: string, sarifToUploadPath: string,
+    ghToken: string, sarifZipPath: string,
     analysisStartTime: string,
     sha: string, ref: string,
     uploadToRepo: { owner: string, repo: string },
+    printReportLink: boolean,
```

Comment:
Since we have output named `report_link`, so this variable name is a bit confusing.
Can we rename it to `printSecurityTabLink`?